calico/operations/ebpf/troubleshoot-ebpf.mdx

```diff
@@ -202,9 +242,89 @@ The parts of the log are explained below:
   - A host endpoint ingress program is executed on the path from external node to the host.
   - A host endpoint egress program is executed on the path from host to external host.
 
+- you may also see `ens192---X` which indicates an XDP program. $[prodname] uses XDP programs to implement `doNotTrack` policies on host devices only.
+
 - `Final result=ALLOW (-1). Program execution time: 7366ns` is the message. In this case, logging the final result of
   the program. Note that the timestamp is massively distorted by the time spent logging.
 
+## Debugging policy issues
+
+If you suspect that $[prodname] is dropping packets due to policy, you can use the
+`calico-node -bpf` tool to dump the policy that is attached to a specific interface.
+
+```
+kubectl exec -n calico-system calico-node-abcdef -- calico-node -bpf policy dump <interface> <hook>
+
+where `<interface>` is the name of the interface and `<hook>` is either `ingress`, `egress`, `xdp` or `all`.
```

Comment:
Move outside the code block, like this:

Replace the following:
* `<interface>`: The name of the interface.
* `<hook>`: description of what this is. Must be one of the following: `ingress`, `egress`, `xdp`, or `all`.